herddb-core/src/main/java/herddb/server/ServerConfiguration.java

```diff
@@ -148,6 +148,9 @@
     public static final String PROPERTY_BOOKKEEPER_LEDGERS_RETENTION_PERIOD = "server.bookkeeper.ledgers.retention.period";
     public static final long PROPERTY_BOOKKEEPER_LEDGERS_RETENTION_PERIOD_DEFAULT = 1000L * 60 * 60 * 24 * 2;
 
+    public static final String PROPERTY_BOOKKEEPER_WAIT_CLUSTER_READY_TIMEOUT = "server.bookkeeper.wait.cluser.ready.timeout";
+    public static final int PROPERTY_BOOKKEEPER_WAIT_CLUSTER_READY_TIMEOUT_DEFAULT = 6000 * 2; // 2 minutes
```

Comment:
did you mean 60_000 * 2? is it in ms right?

Reply:
good catch